### PR TITLE
Re-insert SC tests

### PR DESCRIPTION
**Pull request by alejoe91**
SC tests are suddenly failing.

@yger can you check if they work locally for you with the latest master or this branch?

### CODE REVIEW

**Comment by alejoe91** [comment]
> Back from holidays, I'll catch up with this error. The tests are still failing?

Working now!!! (magically) :)

**Comment by samuelgarcia** [comment]
Don't close it too quickly.
I would courious to see the log when failling before.
@alejoe91 : could we add something when faling to display the error trace (from the shell) when verbose=True ?

File: spikesorters/basesorter.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def run(self, raise_error=True, parallel=False, n_jobs=-1, joblib_backend='loky'
             'sorter_name': str(self.sorter_name),
             'sorter_version': str(self.get_sorter_version()),
             'datetime': now,
-            'runtime_trace':[]
+            'runtime_trace': []
```
Comment (samuelgarcia) [inline spikesorters/basesorter.py:142]: This must be this, certainly.

Comment (samuelgarcia) [reply spikesorters/basesorter.py:142]: good catch.

**Comment by alejoe91** [comment]
> Don't close it too quickly.
> I would courious to see the log when failling before.
> @alejoe91 : could we add something when faling to display the error trace (from the shell) when verbose=True ?

@samuelgarcia I can merge and release a new version so we can release SI as well. We can still investigate why it was failing. What do you think?

**Comment by samuelgarcia** [comment]
OK for merging and releasre.
But just after this could you open a new PR with SC=0.9.9 and trak a potential new fail.

**Comment by alejoe91** [comment]
now SC is 0.9.9

**Pull request merged**